.github/ISSUE_TEMPLATE/01_question.yml

```diff
@@ -0,0 +1,37 @@
+name: Question/Query
```

Comment:
@ronniegeraghty is there some easy way for you to show a screenshot how this is going to render? Maybe you could do it out from your fork? Would help me visualize how our customers will see it. But if it is a hassle then skip.